Scripts/GameManagement/GameManager.gd

```diff
@@ -34,10 +33,16 @@ remotesync func initialize_players():
 		var player = picked_class_scene.instance()
 		player.set_name(str(p))
 		world.add_child(player)
-		player.position = Vector2(randi()%int(get_viewport().size.x), randi()%int(get_viewport().size.y))
+		player.position = random_spawnpoint()
 		players_info[p] = {"position": player.position, "body_rotation": player.get_node("Body").rotation}
 		get_tree().change_scene_to(world)
 		
+func random_spawnpoint():
+	var x_axis = [2054, 750, 3145, 4858, 8208, 11649, 2137, 7730, 5983, 3971, 6275, 9049, 11839, 10444, 9073, 6675, 12331]
```

Comment:
I think better structure to storage those coordinates is one single list with lists inside it. I mean: var cords = [[9,6], [5, 6]]